zrml/swaps/src/benchmarks.rs

```diff
@@ -261,16 +260,8 @@ benchmarks! {
             ScoringRule::CPMM,
             false
         );
-        let pool_amount = T::MinLiquidity::get();
-        let min_assets_out = vec![
-            T::MinLiquidity::get()
-                - bmul(
-                    T::ExitFee::get().saturated_into(),
-                    T::MinLiquidity::get().saturated_into()
-                )?
-                .saturated_into();
-            a as usize
-        ];
+        let pool_amount = T::MinLiquidity::get() / 2u32.into();
+        let min_assets_out = vec![0u32.into(); a as usize];
```

Comment:
Not able to figure out why this change?

Reply:
Yeah, good question!

First of all, the `min_assets_out` parameter has no influence on the benchmarks. So, in order to make sure that no `LimitOut` error is thrown, I just set it to zero. This is simply because its might be somewhat annoying to figure out how much exactly you're getting out, and there's no point, because we're not doing real trading here.

Also, I set the amount of liquidity I'm withdrawing to half of what's in the pool. Before this change, the entire pool got drained, which, given the state of the pool, was not allowed.

Reply:
thanks for explanation!